WINDOWS_INSTALL.md

```diff
@@ -0,0 +1,145 @@
+
+### Additional Information
+
+#### Setting up Access Control
+
+phpGACL access controls are installed and configured automatically during LibreHealth EHR setup. It can be administered within LibreHealth EHR in the `admin` -> `acl` menu. This is a very powerful access control software.
+
+Learn more about phpGACL [here](http://phpgacl.sourceforge.net/). It is recommended to read the phpGACL manual in the `/lh-ehr/Documentation/README.phpgacl.md` file, the online documentation at [LibreHealth](http://librehealth.io/), and the comments in the `/lh-ehr/library/acl.inc` file.
+
+#### Patient Documents Capability
+
+In order to take full advantage of the patient documents capability you must make sure that the configuration settings in `php` file include `file_uploads = On` and that `upload_max_filesize` is appropriate for your use and that `upload_tmp_dir` is set to the correct value that will work on your system.
+
+#### Updating the Project
+
+Be sure to back up your LibreHealthEHR installation and database before updating!
+
+Login to EHR. There should be an orange button at the bottom right which you can hover over to reveal two options. Click the `</>` button that appears. Follow the instructions to update EHR.
+
+### FAQ
+
+**Q. I am getting a `table doesn't exist` error!**
+
+This is because MySQL versions 5.7 and above have `strict mode` on default which needs to be disabled. This can be resolved by making sure Step #7 is fufilled.
+
+**Q. I am getting an `Internal Server Error` when trying to run portal dashboard!**
+
+Open the Apache configuration file and uncomment this line by removing the `#`:
+
+```
+#LoadModule rewrite_module modules/mod_rewrite.so
+```
+
+Search for
+
+```
+AllowOverride None
+```
+
+and change it to
+
+```
+AllowOverride All
+```
+
+**Q. I am getting a `Check that mysqld is running` error when opening up EHR!**
+
+mysqld is a daemon that runs silently and can be invoked to start a mySQL server on your computer. If the credentials that lh-ehr uses to access the mySQL server are incorrect, then mysqld will not initiate the server, resulting in this error. This may have occured if you tried to sync your fork to the upstream repository. It will have reverted the database password in `sqlconf.php` in `C:\xampp\htdocs\lh-ehr\library\`. 
+
+Change the following line to include your own password inside the single quotes.
+
+```
+$pass    = 'libreehr';
+```
+
+Restart Apache Web Server and MySQL.
+
+**Q. How do I get back to the Login Page once I close out?**
+
+Use the login path with localhost. You may also check your browser history. It is recommended to bookmark this site for easy access. Example: [`http://localhost/lh-ehr/interface/login/login.php?site=default`](http://localhost/lh-ehr/interface/login/login.php?site=default)

```

Comment:
I think pointing to `http://localhost/lh-ehr` does the same thing. Takes you to login page if you have the software installed

Reply:
It works. I edited the file.